packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -184,19 +190,21 @@ export const makeScheduler = async (
       return true;
     };
 
-    switch (timeVsSchedule(now, schedule)) {
+    const phase = timeVsSchedule(now, schedule);
+    trace('phase', phase);
```

Comment:
I can imagine running into a problem where we wish we had it, but since we're turning off tracers anyway I agree it's not sufficiently helpful to include